### PR TITLE
Start/Stop Time Entries

### PR DESCRIPTION
**Pull request by willsb**
⚠️  REQUIRES #344 TO BE MERGED BEFORE IT WORKS⚠️ 

Basic start/stop functionality. No animations for the time being, only the bare bones. This will allow us to test stuff like Suggestions/Log much easier.

💞  Squash as soon as tests pass, using the ✨  emoji 💞 

👇  This is what it looks 👇 

And that's the waaaaaaaaay the news goes

### CODE REVIEW

File: Toggl.Daneel/ViewControllers/MainViewController.cs [hunk 1]
```diff
@@ -26,15 +30,47 @@ public override void ViewDidLoad()
...
+            CurrentTimeEntryCard.Layer.BorderWidth = 1;
+            CurrentTimeEntryCard.Layer.CornerRadius = 8;
+            CurrentTimeEntryCard.Layer.BorderColor = Color.TimeEntriesLog.ButtonBorder.ToNativeColor().CGColor;
+            CurrentTimeEntryElapsedTimeLabel.Font = CurrentTimeEntryElapsedTimeLabel.Font.GetMonospacedDigitFont();
+
             StartTimeEntryButton.Transform = CGAffineTransform.MakeScale(0.01f, 0.01f);
             reportsButton.SetImage(UIImage.FromBundle("icReports"), UIControlState.Normal);
             settingsButton.SetImage(UIImage.FromBundle("icSettings"), UIControlState.Normal);
+           
+            var visibility = new MvxVisibilityValueConverter();
```
Comment (arturs105) [inline Toggl.Daneel/ViewControllers/MainViewController.cs]: `visibilityConverter` for consistency

File: Toggl.Daneel/ViewControllers/MainViewController.cs [hunk 2]
```diff
@@ -26,15 +30,47 @@ public override void ViewDidLoad()
...
+            var timeSpanConverter = new TimeSpanToDurationValueConverter();
+            var invertedVisibility = new MvxInvertedVisibilityValueConverter();
```
Comment (arturs105) [inline Toggl.Daneel/ViewControllers/MainViewController.cs]: Same as above

File: Toggl.Daneel/ViewControllers/MainViewController.cs [hunk 3]
```diff
@@ -26,15 +30,47 @@ public override void ViewDidLoad()
...
+                      .For(v => v.BindVisibility())
+                      .To(vm => vm.CurrentlyRunningTimeEntry)
+                      .WithConversion(invertedVisibility);
+
+            //Text
+            bindingSet.Bind(CurrentTimeEntryDescriptionLabel)
+                      .To(vm => vm.CurrentlyRunningTimeEntry.Description);
+
+            bindingSet
+                .Bind(CurrentTimeEntryElapsedTimeLabel)
```
Comment (arturs105) [inline Toggl.Daneel/ViewControllers/MainViewController.cs]: This should be on the same line as `bindingSet` for consistency.

File: Toggl.Daneel/ViewControllers/MainViewController.designer.cs [hunk 4]
```diff
@@ -16,10 +16,30 @@ partial class MainViewController
...
         [GeneratedCode ("iOS Designer", "1.0")]
+        UIKit.UIView CurrentTimeEntryCard { get; set; }
+
+        [Outlet]
+        [GeneratedCode ("iOS Designer", "1.0")]
+        UIKit.UILabel CurrentTimeEntryDescriptionLabel { get; set; }
+
+        [Outlet]
+        [GeneratedCode ("iOS Designer", "1.0")]
+        UIKit.UILabel CurrentTimeEntryElapsedTimeLabel { get; set; }
```
Comment (arturs105) [inline Toggl.Daneel/ViewControllers/MainViewController.designer.cs]: Designs in Zeplin has the minute digits darker than the rest (black vs grey). Was it intentionally left out of this PR?

File: Toggl.Foundation.Tests/MvvmCross/ViewModels/StartTimeEntryViewModelTests.cs [hunk 5]
```diff
@@ -59,5 +61,47 @@ public class TheBackCommand : StartTimeEntryViewModelTest
...
+                var dateParameter = DateParameter.WithDate(date);
+
+                await ViewModel.Initialize(dateParameter);
+                ViewModel.RawTimeEntryText = description;
+                ViewModel.DoneCommand.Execute();
+
+                await DataSource.TimeEntries.Received().Start(
+                    Arg.Is<DateTimeOffset>(receivedDate => receivedDate == dateParameter.GetDate()), 
+                    Arg.Is<string>(receivedDescription => receivedDescription == description),
+                    Arg.Is<bool>(billable => !billable)
```
Comment (arturs105) [inline Toggl.Foundation.Tests/MvvmCross/ViewModels/StartTimeEntryViewModelTests.cs]: I don't understand this. Can't it simply be `true`/`false` or `ViewModel.IsBillable`?

File: Toggl.Daneel/ViewControllers/MainViewController.designer.cs (hunk 4 above)
Comment (willsb) [inline Toggl.Daneel/ViewControllers/MainViewController.designer.cs]: Yes. It'll need a converter that does string to AttributedString conversion. This is really bare bones, so no need for fancy stuff

File: Toggl.Foundation.Tests/MvvmCross/ViewModels/StartTimeEntryViewModelTests.cs (hunk 5 above)
Comment (willsb) [inline Toggl.Foundation.Tests/MvvmCross/ViewModels/StartTimeEntryViewModelTests.cs]: I think that might be possible, let me check

**Pull request merged**